gamemodes/guesswho/gamemode/shared.lua

```diff
@@ -11,6 +11,7 @@ DeriveGamemode( "base" )
 
 include("player_ext_shd.lua")
 include("sh_config.lua")
+include("sh_propfreeze.lua")
```

Comment:
Include in init.lua since script should be serverside only. Also change name to `sv_propfreeze.lua`